openquake/hazardlib/tests/gsim/chiou_youngs_2014_test.py

```diff
@@ -75,8 +76,22 @@ def test_mean_hanging_wall_reversed_slip(self):
     def test_mean_hanging_wall_strike_slip(self):
         self.check('NGA/CY14/CY14_MEDIAN_MS_HW_SS.csv',
                    max_discrep_percentage=0.05)
-    
+
     def test_total_event_stddev(self):
         # Total Sigma fixes at 0.65
         self.check('NGA/CY14/CY14_TOTAL_EVENT_SIGMA_PEER.csv',
                    max_discrep_percentage=0.05)
+
+
+class ChiouYoungs2014NearFaultTestCase(BaseGSIMTestCase):
+    GSIM_CLASS = ChiouYoungs2014NearFaultEffect
+
+    # First five tests use data ported from Kenneth Campbell
```

Comment:
I think this comment need to be updated or removed. It would be useful to know how you generated the verification tables for this case.